src/main/java/com/wire/xenon/models/AudioPreviewMessage.java

```diff
@@ -18,43 +18,12 @@
 
 package com.wire.xenon.models;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
I was confused about this too.
So, DTOs in com.wire.xenon.backend.models and in com.wire.xenon.models.otr are used by Xenon and mostly Helium to actually deserialize responses from RestAPI, so they need Jackson annotations.

The message classes here instead, are only used in a single API exposed only by LegalHold so I had them removed.
Now, while writing this comment I made some considerations and probably it is just easier to keep them like this, instead of handling it later in LH. Put those back and tried to improve deserialization edge cases of QualifiedId